package.json

```diff
@@ -2,9 +2,9 @@
   "name": "angular-sdk",
   "version": "8.23.10",
   "scripts": {
-    "build:prod": "run-p -l lint prod-build-angularsdk",
+    "build:prod": "npm run fix:format && run-p -l lint prod-build-angularsdk",
     "build:prod:ci": "npm run clean && npm install && npm run build:prod",
-    "build:dev": "run-p -l lint build-angularsdk",
+    "build:dev": "npm run fix:format && run-p -l lint build-angularsdk",
```

Comment:
Instead of adding fix:format command, we can change lint as fix command

`"build:dev": "run-p -l fix build-angularsdk",`

Reply:
Hi Vishal, sounds good, I made the change, please approve